bats.mk

```diff
@@ -0,0 +1,62 @@
+WORKSPACE=/workspace
```

Comment:
This file could be moved to our Makefiles.

Reply:
Sure. Do you want to do this with the other changes?